### PR TITLE
feat: Implement mongoose cleaner as npm module

### PR DESCRIPTION
**Pull request by jishi**

### CODE REVIEW

**Comment by jishi** [comment]
What was the purpose for this documentation only PR? Is it for the mobile team?

File: src/mongoose-cleaner.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash';
```
Comment (sommestad) [inline src/mongoose-cleaner.js]: We might want to install and import only `lodash.omit` and `lodash.mapValues` to greatly reduce this module's size?

Comment (sommestad) [inline src/mongoose-cleaner.js]: E.g. https://www.npmjs.com/package/lodash.omit

File: test/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "extends": "springworks/mocha",
+  "globals": {
+    "should": false,
+    "sinon": false
+  },
+  "plugins": [
+    "should-promised"
+  ],
+  "rules": {
+    "springworks/top-describe-path": 2,
```
Comment (sommestad) [inline test/.eslintrc.json]: This is already set as default, right? https://github.com/Springworks/eslint-config-springworks/blob/0a83cf2d4ad784a01e290298bfeade1ea72222bb/mocha-config.json#L10

**Comment by sommestad** [comment]
- [x] We also need the `.npmignore` to ensure we don't publish tests etc, right? (could be separate PR)

File: src/mongoose-cleaner.js (hunk 1 above)
Comment (sommestad) [inline src/mongoose-cleaner.js]: And https://www.npmjs.com/package/lodash.mapvalues

Comment (jishi) [inline src/mongoose-cleaner.js]: We might want to skip lodash altogether, it is fairly trivial to do these two natively.

Comment (sommestad) [inline src/mongoose-cleaner.js]: I'd recommend not reinventing any wheels, even though they're  easy to build. :) We can spend that time on something else. ;)

Comment (jishi) [inline src/mongoose-cleaner.js]: I skipped the omit, seemed unnecessary to clone it twice just to remove a property.

File: test/.eslintrc.json (hunk 2 above)
Comment (jishi) [inline test/.eslintrc.json]: Seems so, I remove it then.

**Pull request merged**